lib/Artwork.php

```diff
@@ -195,14 +195,29 @@ public static function Get(?int $artworkId): Artwork{
 		return $result[0];
 	}
 
+	public static function GetByUrlPath($artistUrlName, $artworkUrlName): Artwork{
```

Comment:
nit: if it can return `null` then the return type of the function should be `?Artwork`